x/concentrated-liquidity/msg_server_test.go

```diff
@@ -226,10 +227,17 @@ func (suite *KeeperTestSuite) TestCollectIncentives_Events() {
 			pool := suite.PrepareConcentratedPool()
 			suite.SetupDefaultPosition(pool.GetId())
 
+			position, err := suite.App.ConcentratedLiquidityKeeper.GetPosition(ctx, tc.positionId)
+			suite.Require().NoError(err)
+
+			ctx = ctx.WithBlockTime(position.JoinTime.Add(time.Hour * 24 * 7))
+
+			positionAge := ctx.BlockTime().Sub(position.JoinTime)
```

Comment:
Can you please explain what's happening here? Also please consider adding comments from your reply

Reply:
Yes, so in order for `CollectInventives` to emit collection events, the position must be old enough to qualify for that incentive. This is just setting up the position, and then increasing the block time 1 week in the future. I added in line comments here

https://github.com/osmosis-labs/osmosis/pull/4724/commits/6266ec5e3a186b4956c82bf0350bfb279308a40a